src/css/app.css

```diff
@@ -1,38 +1,44 @@
 /* app global css */
 
+:root {
+    --fgcolor: #efefef;
+    --bgcolor: #303030;
+}
+
 #app {
-    background-color: tan;
+    background-color: var(--bgcolor, tan);
```

Comment:
This construct uses the --bgcolor variable if it has been defined, or else uses the tan variable as fallback